WordPress/src/main/java/org/wordpress/android/ui/stats/JetpackConnectionDeeplinkActivity.java

```diff
@@ -37,8 +38,9 @@
  * <p>
  * Redirects users to the stats activity if the jetpack connection was succesful
  */
-public class StatsDeeplinkActivity extends AppCompatActivity {
+public class JetpackConnectionDeeplinkActivity extends AppCompatActivity {
```

Comment:
Do you think we should move this out of the `org.wordpress.android.ui.stats` package since it's no longer just for stats?